src/IssueViz.Security/Hotspots/HotspotReviewPriorityProvider.cs

```diff
@@ -0,0 +1,222 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using SonarLint.VisualStudio.IssueVisualization.Security.Hotspots.Models;
+
+namespace SonarLint.VisualStudio.IssueVisualization.Security.Hotspots
+{
+    /// <summary>
+    /// Provides a mapping for hotspots to their review priority
+    /// </summary>
+    public interface IHotspotReviewPriorityProvider
+    {
+        /// <summary>
+        /// Supported languages:
+        /// <list type="bullet">
+        /// <item><description>JavaScript</description></item>
+        /// <item><description>TypeScript</description></item>
+        /// <item><description>C++</description></item>
+        /// <item><description>C</description></item>
+        /// </list>
+        /// </summary>
+        /// <param name="hotspotKey">Hotspot Rule Key</param>
+        /// <returns>Associated review priority or null if not mapped</returns>
+        HotspotPriority? GetPriority(string hotspotKey);
+    }
+
+    [Export(typeof(IHotspotReviewPriorityProvider))]
+    [PartCreationPolicy(CreationPolicy.Shared)]
+    public class HotspotReviewPriorityProvider : IHotspotReviewPriorityProvider
+    {
+        private static readonly Dictionary<string, HotspotPriority> KeyToPriorityMap = new Dictionary<string, HotspotPriority>
+        {
+            { "cpp:S5443", HotspotPriority.Low },
```

Comment:
It would be useful if we had a test that checked that no new hotspots have been added.
However, we don't need it for this sprint. Could you add an Infra ticket to the backlog please(/

Reply:
https://github.com/SonarSource/sonarlint-visualstudio/issues/4554